save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -130,7 +131,7 @@ external interface ProjectViewState : RState {
 @JsExport
 @Suppress("CUSTOM_GETTERS_SETTERS")
 class ProjectView : RComponent<ProjectExecutionRouteProps, ProjectViewState>() {
-    private val testTypesList = listOf("Class", "Comment", "Function", "Variable", "Enum", "Space", "Style", "Another")
+    private var testTypesList: List<TestSuiteDto> = listOf()
```

Comment:
I believe we had a rule in diktat, that should've suggested to use `emptyList` instead of `listOf()`?

Reply:
Please check if it's working in diktat